src/XMLimport.cpp

```diff
@@ -872,18 +851,8 @@ void XMLimport::readHostPackage(Host* pHost)
     } else {
         pHost->mRequiredDiscordUserDiscriminator.clear();
     }
-
-    if (attributes().hasAttribute(QLatin1String("mSGRCodeHasColSpaceId"))) {
-        pHost->setHaveColorSpaceId(attributes().value(QLatin1String("mSGRCodeHasColSpaceId")).toString() == QLatin1String("yes"));
-    } else {
-        pHost->setHaveColorSpaceId(false);
-    }
-
-    if (attributes().hasAttribute(QLatin1String("mServerMayRedefineColors"))) {
-        pHost->setMayRedefineColors(attributes().value(QLatin1String("mServerMayRedefineColors")).toString() == QLatin1String("yes"));
-    } else {
-        pHost->setMayRedefineColors(false);
-    }
+    pHost->setHaveColorSpaceId(attributes().value(QLatin1String("mSGRCodeHasColSpaceId")).toString() == QLatin1String("yes"));
+    pHost->setMayRedefineColors(attributes().value(QLatin1String("mServerMayRedefineColors")).toString() == QLatin1String("yes"));
```

Comment:
Do these still work if there is no attribute of the given name for `value(...)` to return - i.e. that returns an empty `QStringRef` - reading through that class's documentation suggests that some of it's methods do not behave well with an empty string...

Reply:
Not sure about it. If you're asking if it works - yes it works. If you're asking what will happen if we'll pass empty string, no idea. Not the case here. If you think this should be replaced with `QStringLiteral` as well I can do it.

Reply:
> ... If you're asking what will happen if we'll pass empty string, no idea. ...

It is just that for an old save from a while ago won't have some attributes and in that case `attributes().value(QLatin1String("someNewSettingThatIsNotInOldSaves")` will return an **empty** `QStringRef` and I am not 100% certain what `(QString) QStringRef::toString()` will do in that case given that some of the other methods in that class promise *undefined behaviour* (i.e. straying in **[`nasal-daemons`](https://en.wikipedia.org/wiki/Nasal_demons)** territory) in that situation...

Reply:
So is there any real benefit of using `QLatin1String` here? For simplicity maybe better use `QStringLieral`